examples/streaming/knative-eventing/assets/event-display-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+
+# event-display app deploment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: event-display
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
```

Comment:
That is very cool! Would that just match whatever is on `spec.template.metadata.labels`?

Reply:
Yeah! I didn't even know that was a thing, but it is!